### ISSUE TITLE
[FR] Add a setting for keeping delimeter

### ISSUE THREAD

**Issue by emisjerry** [trey-wallis/obsidian-note-splitter#13]
i have a note:

```
In the city zoo, there were two sections. One was the clean section and the
other was the dirty section. 
```

The delimeter is the sentence-ending dot (.). Could you have a setting for keeping the ending dot?

My requirement: after splitting:

Note 1
```
In the city zoo, there were two sections. 
```

Note 2
```
One was the clean section and the other was the dirty section. 
```

Thanks.

**Comment by emisjerry** [comment]
BTW,I have a YouTube video that uses Note Splitter, maybe you'd be interested in seeing it:

https://youtu.be/Gu6B7nqUV9o

**Comment by decaf-dev** [comment]
Hey emisjerry. I will add this feature today.

**Comment by decaf-dev** [comment]
Wow! The video is great. I'm glad you're finding good use of the plugin.

It seems like "Note Splitter" doesn't translate well into Chinese. But your
pronunciation is very good!!

**Comment by decaf-dev** [comment]
@emisjerry Added in 1.2.0.

The way I ended up implementing this behavior was adding an *Append to split content* setting. I thought this would be the best option for normal use cases. Please check the updated README for an explanation of the setting: https://github.com/decaf-dev/obsidian-note-splitter?tab=readme-ov-file#append-to-split-content. Cheers!

**Comment by emisjerry** [comment]
Thanks. I'll try it ASAP.

**Comment by decaf-dev** [comment]
@emisjerry Would you mind if I put a link to your video into the README documentation?

**Comment by emisjerry** [comment]
Sure. It's my honor.

**Comment by decaf-dev** [comment]
@emisjerry I just added a new setting called `Remove delimiter`. This is probably a more intuitive way to enable the behavior that you were describing about keeping the period in your split notes. This is now released in 1.3.0.
